packages/suite/src/types/utils/index.ts

```diff
@@ -38,3 +40,8 @@ export type DeepPartial<T> = T extends () => any
     : T extends { [key: string]: any }
     ? { [P in keyof T]?: DeepPartial<T[P]> }
     : T;
+
+// Type the yup schema shape
+export type Shape<T> = Record<keyof T, AnySchema>;
```

Comment:
Shape is too much generic without context, use something more specific related to validation ValidationShape, ValidationSchema, YupSchema...

Reply:
done